README.md

```diff
@@ -82,14 +82,15 @@ KEEP_MY_INTERNAL_POSTGRES = False
 ### 4. Create a notebook in IBM Watson Studio
 
 * In [Watson Studio](https://dataplatform.cloud.ibm.com/), click `New Project +` under Projects or, at the top of the page click `+ New` and choose the tile for `Data Science` and then `Create Project`.
-* In your project go to the `Settings` tab, scroll down to `Associated Services` and choose `+ Add service` -> `Spark`. Either choose and `Existing` Spark service, or create a `New` one.
 * In [Watson Studio](https://dataplatform.cloud.ibm.com/) using the project you've created, click on `+ Add to project` and then choose the  `Notebook` tile, OR in the `Assets` tab under `Notebooks` choose `+ New notebook` to create a notebook.
-* Select the `From URL` tab.
-* Enter a name for the notebook.
-* Optionally, enter a description for the notebook.
-* Under `Notebook URL` provide the following url: https://raw.githubusercontent.com/IBM/monitor-wml-model-with-watson-openscale/master/notebooks/OpenScale.ipynb
-* Select the Spark runtime you've associated with this project:
-* Click the `Create` button.
+* Select the `From URL` tab. [1]
```

Comment:
I like the numbering.  I wonder if it would be better to show the pictures first so you are not referencing something they haven't seen yet.